api/envoy/config/rbac/v3/rbac.proto

```diff
@@ -68,19 +74,28 @@ message RBAC {
 
   // Should we do safe-list or block-list style access control?
   enum Action {
-    // The policies grant access to principals. The rest is denied. This is safe-list style
+    // The policies grant access to principals. The rest are denied. This is safe-list style
     // access control. This is the default type.
     ALLOW = 0;
 
-    // The policies deny access to principals. The rest is allowed. This is block-list style
+    // The policies deny access to principals. The rest are allowed. This is block-list style
     // access control.
     DENY = 1;
+
+    // The policies set the `access_log_hint` dynamic metadata key based on if requests match
+    LOG = 2;
   }
 
-  // The action to take if a policy matches. The request is allowed if and only if:
+  // The action to take if a policy matches.
+  // The request is allowed if and only if:
   //
-  //   * `action` is "ALLOWED" and at least one policy matches
+  //   * `action` is "ALLOW" and at least one policy matches
   //   * `action` is "DENY" and none of the policies match
+  //   * `action` is "LOG"
```

Comment:
I think this is still a bit misleading, reading this it looks like the request is allowed if the action is LOG and the the policy matched? The LOG action doesn't decide allow or deny a request.

Also what happens if there are only LOG action in the filter config?

Reply:
I thought it makes sense, if the action is LOG it's not qualified by any policies. So like you say the action is always allowed when it is LOG. I'll refactor this to make it more clear though.